less/home-page.less

```diff
@@ -72,4 +80,60 @@
       }
     }
   }
+
+  @media @phone {
```

Comment:
this is cool but I'd like to point out that you could create media queries for each element much easier now with less